src/todoListItem.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+function TodoListItem(props){
+    return (
```

Comment:
Nitpick (doesn’t require changes):
The wrapping `div` isn’t needed because the `li` element contains all the expressions/values within its tags. You’d need a wrapping parent element if you have expressions/values/html that are defined outside of the li element:

```jsx
<>
  <p>hello</p>
  <p>world</p>
</>
```
The above example uses a React fragment (this is a more elegant way to wrap two or more elements without adding an unnecessary element in the DOM that is defined just to contain multiple elements — that said, if you wanted to a class to it, then it’d need to be a valid HTML element) to wrap the hello element and world element, so that both elements share a parent wrapping element.